### PR TITLE
KBI on addurls for tracking simple collections of files

### PR DESCRIPTION
**Pull request by mih**

### CODE REVIEW

File: kbi/0007/index.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+downloaded -- with or without requiring authentication. When such data needs to
+be processed, or ingested into another system or database, it can make sense to
+use a DataLad dataset as means to track the exact state of any downloaded data
+and thereby define a precise starting point for further processing or
+reporting. This is particularly relevant for data files that may change over
+time.
+
+This KBI describes a simple approach how any collection of files that are
+accessible via some URL can be tracked in a DataLad dataset. This approach is
+applicable to a large number of data sources, such as the `Eurostat`_ (the EU's
```
Comment (adswa) [inline kbi/0007/index.rst:21]:
```suggestion
applicable to a large number of data sources, such as the `Eurostat`_ portal (the EU's
```

File: kbi/0007/index.rst [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+.. code-block:: console
+
+   % datalad create gse209630
+   create(ok): /tmp/gse209630 (dataset)
+
+We use the ``create`` command and the accession number for the name of the directory that shall contain the newly created DataLad dataset.
+
+Next we can use the ``addurls`` command to populate the dataset with the files
+given in the table. The command requires three parameters: 1) the path to the
+CSV table; 2) the name of the table column with the access URL of a file; and
```
Comment (adswa) [inline kbi/0007/index.rst:76]:
```suggestion
CSV table; 2) the name of the table column with the access URL of a file enclosed in curly braces (``{}``); and
```

File: kbi/0007/index.rst [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+3) the name of the table column with the associated file name to be used in the
+DataLad dataset.
```
Comment (adswa) [inline kbi/0007/index.rst:78]:
```suggestion
3) the name of the table column with the associated file name to be used in the
DataLad dataset enclosed in curly braces (``{}``).
```

**Review by adswa — APPROVED** [review]
left a few suggestions, otherwise it looks good to merge to me

File: kbi/0007/index.rst (hunk 2 above)
Comment (mih) [reply kbi/0007/index.rst:76]: Thanks, I addressed this too with a small edit of your following suggestion.

**Pull request merged**